data/Trikcode.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "Nobert Patrick",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Full Stack Developer | **Open Source Contributor**",
+  "avatar": "https://github.com/Trikcode.png",
+  "tags": [
+			"Python",
+			".NET/C#",
+			"Javascript",			
+			"Typescript",
+			"Open Source"
```

These tags look great, but there seems to be too much indentation here. Shifting them out a little will look great.
```suggestion
  "tags": [
        "Python",
	".NET/C#",
	"Javascript",			
	"Typescript",
	"Open Source"
```

---

Thanks so much I have updated the file